ChartIQ/Charts/ChartIQView.swift

```diff
@@ -653,10 +652,11 @@ public class ChartIQView: UIView {
     public func push(_ data: [ChartIQData]) {
         let obj = data.map{ $0.toDictionary() }
         let jsonData = try! JSONSerialization.data(withJSONObject: obj, options: .prettyPrinted)
-        let jsonString = String(data: jsonData, encoding: .utf8)
-        let script =
-            "callNewChart(\"\", \(jsonString!)); "
-        webView.evaluateJavaScript(script, completionHandler: nil)
+        let jsonString = String(data: jsonData, encoding: .utf8)?.replacingOccurrences(of: "\n", with: "") ?? ""
+        let script = "callNewChart(\"\", \(jsonString)); "
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```

Comment:
Now that you switched the order in the ViewController, do you still need this delay?

Reply:
Unfortunately yes. After testing this didn't fix the overall push issue. Another ticket will have to be created to do deeper investigation.

I thought I left a comment about this earlier, guess it didn't save.

Reply:
What you changed in ViewController is for pull.  This function deals with push.